data_models/gcp_data_model.py

```diff
@@ -38,12 +37,14 @@ def get_admin_map(event):
 
 
 def get_modified_users(event):
-    roles_assigned = get_admin_map(event)
+    event_dict = event.to_dict()
+    roles_assigned = get_admin_map(event_dict)
 
-    return json.dumps(list(roles_assigned.keys()), default=PantherEvent.json_encoder)
+    return json.dumps(list(roles_assigned.keys()))
 
 
 def get_iam_roles(event):
-    roles_assigned = get_admin_map(event)
+    event_dict = event.to_dict()
+    roles_assigned = get_admin_map(event_dict)
 
-    return json.dumps(list(roles_assigned.values()), default=PantherEvent.json_encoder)
+    return json.dumps(list(roles_assigned.values()))
```

Comment:
I'd feel a lot better about this if we had test cases for this.  I'm used to seeing overwrites of the default marshaler for `datetime.datetime` objects in python.. do we know with certainty that the values will be marshalable?

Reply:
I know this doesn't address your core concern, but this is tested tangentially by detections that use this data model.  As for timestamps, you expect that the roles_assigned could have a timestamp in them? Can you point me to the error you have been seeing?

Reply:
The testing suite sends all datetimes in as strings, so this issue wouldn't surface there. I followed the history of this addition to the file, and there's no indication that any problem was being overcome with by overriding the marshaler 🤷 . It's probably fine as long as we don't try to interact with `event['timestamp']`